package.json

```diff
@@ -13,6 +13,7 @@
     "@commitlint/cli": "^6.1.3",
     "@commitlint/config-conventional": "^6.1.3",
     "@commitlint/config-lerna-scopes": "^6.1.3",
+    "@commitlint/travis-cli": "^6.1.3",
```

Comment:
+1 for using an existing npm package instead of maintaining our own linter script 👏